cmd/updatehub/main.go

```diff
@@ -132,3 +140,23 @@ func main() {
 
 	os.Exit(d.Run())
 }
+
+func sanitizeServerAddress(address string, disableHTTPS bool) (string, error) {
+	a := address
+	if !strings.HasPrefix(a, "http://") && !strings.HasPrefix(a, "https://") {
+		a = "https://" + a
```

Comment:
@giulianvivan I believe the prefix should be changed depending on disable-ssl option ... or am I missing something?